### ISSUE TITLE
conflict with Essential Addons for Elementor

### ISSUE THREAD

**Issue by SusanTapper** [weDevsOfficial/happy-elementor-addons#4]
when I uploaded Happy Addons for Elementor I couldn't edit the Data Table that I created with Essential Addons for Elementor.

**Comment by obiPlabon** [comment]
Hello @SusanTapper could you please help us to resolve the situation you faced by providing more information?

1. What's your Essential Addons version
2. What's your HappyAddons version
3. Did you see any error on your browser console? Could you please send us a screenshot?

Thanks

**Comment by SusanTapper** [comment]
Hi Support,

Thank you for getting back in touch with me.

Below are two screen shots.

This is the screen before clicking on the table.

  <https://d33v4339jhl8k0.cloudfront.net/inline/88230/3c0fa29c4f47738450ae3a82cbedf0d480da8cbe/e3601a715789ffdf6b8e9ff78e020a4848e6ff50/image005.jpg> 

This is the screen after clicking on the table, as you see on the left hand side, no options for me to edit.

  <https://d33v4339jhl8k0.cloudfront.net/inline/88230/17a8a8126325341955cff90ace79c29f29f345e5/6944d4b74c6146cc8d7039dd73531075bc8d9007/image006.jpg> 




I was able to fix the issue : I had to disable the Happy Addons for Elementor plugin, make my changes in EA Data Table and then Activate the Happy plugin again.

 

The Versions are :

*	Essential Addons for Elementor Version 3.2.1
*	Happy Elementor Addons Version 1.5.0

 

There were NO error messages to alert me of the incompatibility. 

 

Regards,

Susan Tapper

Owner | Website Designer | Feng Shui Consultant

The Esplanade, Warners Bay NSW 2282

' 0400 366 156 | *  <mailto:contact@webtappers.com> contact@webtappers.com | * www.webtappers.com
